src/api/kube.ts

```diff
@@ -0,0 +1,397 @@
+// tslint:disable:object-curly-spacing
+
+import { Apps_v1Api, Core_v1Api, Extensions_v1beta1Api, KubeConfig, RbacAuthorization_v1Api, V1ConfigMap, V1ConfigMapEnvSource, V1Container, V1DeleteOptions, V1Deployment, V1DeploymentSpec, V1EnvFromSource, V1LabelSelector, V1ObjectMeta, V1Pod, V1PodSpec, V1PodTemplateSpec, V1RoleBinding, V1RoleRef, V1ServiceAccount, V1Subject } from '@kubernetes/client-node'
+import { cli } from 'cli-ux'
+import { readFileSync } from 'fs'
+import * as yaml from 'js-yaml'
+
+export class KubeHelper {
+  kc = new KubeConfig()
+
+  constructor(context?: string) {
+    if (!context) {
+      this.kc.loadFromDefault()
+    } else {
+      this.kc.loadFromString(context)
+    }
+  }
+
+  async serviceAccountExist(name: string | undefined = '', namespace: string | undefined = ''): Promise<boolean> {
+    const k8sApi = this.kc.makeApiClient(Core_v1Api)
+    try {
+      const res = await k8sApi.readNamespacedServiceAccount(name, namespace)
+      if (res && res.body &&
+        res.body.metadata && res.body.metadata.name
+        && res.body.metadata.name === name) {
+        return true
+      } else {
+        return false
+      }
+    } catch {
+      return false
+    }
+  }
+
+  async createServiceAccount(name: string | undefined = '', namespace: string | undefined = '') {
+    const k8sCoreApi = this.kc.makeApiClient(Core_v1Api)
+    let sa = new V1ServiceAccount()
+    sa.metadata = new V1ObjectMeta()
+    sa.metadata.name = name
+    sa.metadata.namespace = namespace
+    try {
+      const response = await k8sCoreApi.createNamespacedServiceAccount(namespace, sa)
+      return response
```

Comment:
Looks like the `response` constant is redundant. Here and below.

Reply:
+1